src/definitions/elements/label.less

```diff
@@ -385,6 +385,26 @@ a.ui.label {
   padding-left: @ribbonTriangleSize;
   padding-right: @ribbonDistance;
 }
+.ui.basic.ribbon.label {
+  padding-top: @basicRibbonOffset;
+  padding-bottom: @basicRibbonOffset;
+}
+.ui.basic.ribbon.label:not([class*="right ribbon"]) {
+  padding-left: @basicRibbonTriangleSizeOffset;
+  padding-right: @basicRibbonTriangleSize;
+}
+.ui.basic.ribbon.label:not([class*="right ribbon"])::after {
+  top: e(%("calc(100%% + %d)", @basicBorderWidth));
```

Comment:
please move this into a variable as well to stay consistent (even if only used once) thanks

Reply:
[label.less#L258](https://github.com/fomantic/Fomantic-UI/blob/master/src/definitions/elements/label.less#L258) has the same expression as my proposed addition:

`@basicRibbonAfterTopOffset: e(%("calc(100%% + %d)", @basicBorderWidth));`

Could they be combined? Perhaps, as @basicLabelOffset or something...

Reply:
Good catch! Yes, you can combine them as you suggested 👍

Reply:
Combined as @basicBorderFullWidthOffset